verification/benchmark_username_gen.php

```
@@ -0,0 +1,135 @@
+        // Simplified mock: if query contains LIKE 'testuser%', return all matches
+        if (strpos($query, "LIKE 'testuser%'") !== false) {
+             $matches = [];
+             foreach ($existing_usernames as $u) {
+                 if (stripos($u, 'testuser') === 0) {
+                     $matches[] = $u;
+                 }
+             }
+             return $matches;
+        }
+        return [];
+    }
```

PHPMD (2.15.0)

[error] The method get_col is not named in camelCase. (undefined)

(CamelCaseMethodName)